File: 2076-sum-of-digits-of-string-after-convert/solution.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        string a="";
        for(int i=0;i<s.length();i++){
            a+=to_string(s[i]-'a'+1);
        }
        cout<<a<<" ";
        for(int i=0;i<k;i++){
            int c=0;
            for(char v:a){
                c+=v-'1'+1;
            }
            a=to_string(c);
        }
        cout<<a;
        return stoi(a);
    }
};

File: 2076-sum-of-digits-of-string-after-convert/solution.java
class Solution {
    public int getLucky(String s, int k) {
        String ans="";
        for(int i=0;i<s.length();i++){
            ans+=((s.charAt(i)-'a'+1));
        }
        int loop=0;
        while(loop<k){
            int sum=0;
            for(int i=0;i<ans.length();i++){
            sum+=(ans.charAt(i)-'0');
            }
            ans=Integer.toString(sum);
            loop++;
        }
        return Integer.parseInt(ans);
    }
}
